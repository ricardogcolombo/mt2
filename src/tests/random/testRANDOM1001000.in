100 1000
1 17 102 40 53
1 19 69 51 111
1 16 109 22 82
1 43 61 80 69
1 27 76 89 12
1 31 38 6 52
1 27 51 56 45
1 27 1 62 127
1 39 80 54 115
1 22 81 13 58
1 97 27 11 43
1 75 28 42 7
1 1 121 24 84
1 48 17 73 121
1 98 65 82 97
1 97 94 72 20
1 70 122 54 5
1 58 85 72 35
1 92 25 75 36
1 64 98 90 108
1 53 82 89 20
1 63 111 96 73
1 47 9 76 18
1 70 50 29 18
1 70 50 54 57
1 43 103 75 0
1 14 94 1 55
1 44 26 52 32
1 18 77 58 31
1 58 32 97 43
1 37 43 36 100
1 97 25 96 40
1 14 49 90 16
1 70 58 48 79
1 14 47 47 129
1 88 95 40 128
1 69 46 13 100
1 68 104 75 71
1 68 9 78 4
1 49 127 40 27
1 88 43 82 115
1 29 42 48 73
1 7 24 71 51
1 20 46 87 8
1 29 125 35 119
1 59 74 86 65
1 72 104 85 30
1 15 4 3 49
1 68 37 60 60
1 22 89 99 98
1 87 42 8 4
1 74 31 2 114
1 17 112 24 62
1 56 79 100 121
1 63 34 67 77
1 93 59 40 95
1 84 121 71 92
1 14 11 98 38
1 25 25 51 48
1 26 111 16 40
1 4 21 98 20
1 60 24 56 68
1 34 5 12 107
1 9 93 70 115
1 4 9 44 47
1 97 78 72 109
1 47 126 67 90
1 33 110 65 71
1 23 20 7 95
1 92 47 3 119
1 95 114 23 56
1 96 119 49 85
1 36 28 64 16
1 3 30 54 96
1 84 120 1 48
1 8 34 34 35
1 83 128 11 116
1 82 91 55 81
1 86 109 2 4
1 43 102 26 37
1 7 75 81 16
1 58 32 30 59
1 58 88 19 101
1 12 119 79 94
1 24 59 100 67
1 21 130 10 126
1 83 98 65 110
1 34 38 7 15
1 57 99 15 92
1 35 96 51 71
1 67 47 80 56
1 99 51 18 94
1 34 52 5 46
1 55 25 67 60
1 5 60 70 100
1 44 108 15 112
1 19 10 70 126
1 66 35 21 16
1 17 96 6 52
1 22 10 72 67
1 16 99 22 59
1 68 28 95 59
1 46 75 12 126
1 64 63 83 115
1 9 100 69 52
1 19 57 56 42
1 79 83 25 61
1 11 31 87 28
1 43 116 19 83
1 78 96 58 54
1 2 22 9 78
1 3 14 22 49
1 11 122 80 64
1 49 126 61 87
1 67 10 69 4
1 76 61 34 36
1 33 63 19 114
1 34 35 12 91
1 95 40 96 70
1 66 84 52 3
1 43 103 34 83
1 32 19 61 77
1 53 82 28 68
1 38 17 14 116
1 80 112 99 36
1 90 37 86 29
1 52 24 30 37
1 15 126 20 95
1 21 89 76 36
1 39 9 60 116
1 90 55 97 91
1 81 25 31 71
1 51 85 58 77
1 34 75 95 31
1 18 97 92 43
1 73 93 56 46
1 43 7 10 59
1 44 85 78 35
1 14 42 51 28
1 82 22 7 77
1 88 117 65 61
1 12 25 78 86
1 2 15 44 61
1 32 32 72 43
1 7 32 86 46
1 88 104 79 80
1 57 27 90 46
1 31 23 71 60
1 67 107 95 122
1 44 99 13 22
1 50 69 60 119
1 79 106 2 32
1 66 114 80 121
1 56 33 91 101
1 14 43 77 90
1 74 16 85 13
1 85 29 58 62
1 73 59 62 101
1 41 75 18 7
1 70 103 22 28
1 88 8 33 122
1 20 102 72 105
1 77 65 93 29
1 74 79 42 87
1 98 0 82 71
1 95 8 59 0
1 42 67 41 42
1 29 6 68 16
1 57 89 85 52
1 3 39 86 65
1 68 91 1 40
1 59 23 49 102
1 14 15 21 57
1 68 81 22 12
1 28 115 61 19
1 47 58 61 22
1 17 57 88 79
1 44 41 74 93
1 80 67 97 7
1 59 1 77 57
1 82 90 71 9
1 2 99 15 62
1 51 59 24 45
1 70 128 66 65
1 36 57 44 20
1 79 57 48 44
1 40 48 77 78
1 87 98 69 91
1 77 125 62 14
1 64 56 63 91
1 68 37 35 117
1 87 17 43 36
1 84 40 22 70
1 94 16 74 28
1 95 62 90 75
1 62 48 12 1
1 16 23 22 12
1 44 10 54 25
1 91 85 53 113
1 55 58 1 46
1 24 50 7 81
1 53 91 12 17
1 90 5 68 33
1 21 54 30 80
1 27 6 100 46
1 96 52 44 107
1 30 62 54 105
1 100 100 30 120
1 54 74 14 4
1 34 124 35 70
1 50 105 74 109
1 60 43 34 18
1 29 54 80 107
1 11 121 8 117
1 79 55 21 110
1 67 20 27 47
1 37 120 68 69
1 91 8 76 80
1 26 57 49 43
1 25 41 96 23
1 54 31 37 97
1 66 18 39 61
1 62 54 66 115
1 88 28 15 118
1 3 120 16 87
1 70 74 6 53
1 55 35 90 11
1 9 124 91 35
1 21 72 66 13
1 28 11 70 112
1 49 114 39 15
1 37 17 46 38
1 84 30 2 42
1 59 20 92 18
1 46 106 11 65
1 70 47 53 123
1 54 100 1 20
1 62 59 40 110
1 53 98 64 89
1 82 7 27 87
1 6 72 77 128
1 41 94 3 76
1 64 63 99 28
1 78 67 62 85
1 47 97 81 98
1 19 4 9 60
1 68 81 98 39
1 58 96 77 82
1 68 130 36 37
1 47 85 7 100
1 52 117 5 53
1 98 6 96 10
1 42 81 54 24
1 20 12 16 103
1 88 26 15 40
1 61 27 35 19
1 87 130 65 75
1 86 22 85 70
1 36 36 82 126
1 25 48 35 75
1 8 43 84 28
1 64 64 90 2
1 36 2 65 115
1 5 58 72 52
1 69 97 43 127
1 26 71 70 79
1 24 27 39 110
1 82 96 94 102
1 4 11 52 84
1 65 60 88 79
1 57 21 13 111
1 50 18 6 102
1 94 23 86 102
1 46 102 74 87
1 100 36 46 100
1 48 128 84 69
1 4 111 3 83
1 88 14 9 98
1 8 105 49 87
1 76 60 81 71
1 11 12 15 14
1 99 127 39 69
1 52 51 4 37
1 60 11 83 43
1 86 21 76 25
1 97 42 24 18
1 5 117 72 66
1 89 117 88 54
1 8 3 82 14
1 76 15 84 13
1 33 89 14 70
1 97 111 45 54
1 87 35 22 20
1 84 104 36 66
1 44 18 45 22
1 49 17 54 44
1 63 16 93 78
1 6 130 97 7
1 46 74 12 78
1 58 54 96 11
1 80 7 86 49
1 29 17 69 19
1 76 95 5 62
1 22 96 88 63
1 11 76 44 121
1 85 67 92 100
1 5 35 38 116
1 7 117 76 102
1 70 78 40 14
1 74 118 81 18
1 63 43 29 22
1 29 120 9 123
1 63 107 77 110
1 20 41 46 75
1 89 63 10 58
1 37 102 56 38
1 72 116 66 101
1 10 52 65 34
1 35 50 48 85
1 25 91 26 7
1 38 52 61 59
1 62 127 4 7
1 80 28 86 84
1 5 19 39 44
1 15 3 27 42
1 58 98 41 21
1 93 119 60 106
1 75 15 18 78
1 54 92 81 14
1 95 88 53 92
1 18 43 67 103
1 77 39 54 106
1 99 101 97 33
1 25 59 63 108
1 70 75 89 34
1 79 116 48 10
1 91 59 9 99
1 43 52 57 21
1 64 66 55 43
1 91 97 34 49
1 81 35 40 76
1 20 29 75 54
1 7 85 100 59
1 35 129 17 107
1 22 101 66 89
1 16 32 4 23
1 69 87 28 92
1 97 81 40 115
1 81 89 93 129
1 25 34 100 6
1 90 4 20 2
1 77 85 75 28
1 53 127 29 102
1 38 70 11 103
1 2 49 41 42
1 60 115 96 120
1 81 4 49 65
1 87 86 27 29
1 76 127 45 59
1 91 44 12 3
1 86 3 47 101
1 15 70 88 22
1 100 117 24 128
1 41 121 50 25
1 1 106 41 112
1 7 47 39 122
1 80 115 33 103
1 68 16 19 54
1 96 123 43 92
1 43 117 53 127
1 42 42 15 6
1 89 100 77 62
1 4 116 59 4
1 18 56 42 100
1 81 45 12 71
1 86 38 40 128
1 3 49 78 59
1 58 64 55 52
1 10 55 36 126
1 65 37 92 29
1 67 121 65 100
1 78 125 42 49
1 88 75 17 90
1 96 37 65 65
1 10 44 63 89
1 54 80 6 98
1 47 104 90 16
1 43 79 61 64
1 68 103 89 120
1 22 78 79 90
1 22 45 37 117
1 54 98 31 36
1 98 109 41 27
1 97 43 66 108
1 18 46 71 65
1 13 62 51 84
1 86 85 71 123
1 39 4 26 40
1 2 57 82 89
1 57 38 9 30
1 3 114 34 99
1 66 108 22 8
1 71 123 89 121
1 42 59 48 30
1 4 12 82 9
1 60 20 56 66
1 78 68 83 39
1 64 1 93 22
1 6 101 62 18
1 22 70 33 67
1 19 93 79 69
1 65 97 86 34
1 64 31 30 28
1 3 120 82 42
1 11 73 48 48
1 29 60 38 81
1 30 52 15 13
1 3 29 40 102
1 91 98 48 25
1 87 44 80 51
1 70 77 62 28
1 68 83 88 65
1 72 39 28 85
1 69 3 85 18
1 1 98 61 112
1 73 45 63 18
1 98 56 64 82
1 70 87 53 83
1 71 74 70 129
1 70 126 52 93
1 68 65 24 61
1 94 41 36 24
1 38 117 5 40
1 57 13 60 58
1 13 49 1 7
1 66 54 19 101
1 68 34 29 126
1 58 130 13 61
1 6 5 92 46
1 27 30 46 90
1 77 25 55 88
1 9 9 88 100
1 26 119 58 130
1 30 13 5 30
1 74 84 10 33
1 73 80 26 18
1 26 90 90 33
1 60 103 36 110
1 50 98 74 89
1 72 0 69 96
1 58 99 79 46
1 11 130 23 96
1 43 7 84 3
1 2 43 61 114
1 84 49 34 107
1 62 12 42 53
1 28 127 2 5
1 24 78 37 117
1 82 65 96 38
1 40 127 43 48
1 6 83 18 20
1 37 82 1 59
1 94 119 76 112
1 65 27 14 4
1 61 95 5 128
1 24 76 93 12
1 64 57 17 121
1 92 33 45 86
1 15 6 21 58
1 29 28 11 48
1 45 108 41 9
1 48 30 85 44
1 35 89 2 122
1 36 112 91 18
1 64 19 94 77
1 31 91 84 75
1 29 56 52 73
1 87 90 49 6
1 57 109 1 120
1 70 49 8 95
1 75 54 36 37
1 49 92 11 119
1 85 73 61 87
1 76 19 100 4
1 79 104 23 33
1 11 99 79 127
1 44 2 83 121
1 21 0 38 39
1 58 31 62 19
1 79 94 45 32
1 14 111 54 64
1 13 33 78 122
1 21 64 24 19
1 5 117 96 97
1 95 39 22 4
1 82 34 92 24
1 45 117 20 40
1 54 22 48 96
1 83 67 88 110
1 45 12 71 89
1 24 7 99 13
1 35 43 83 15
1 16 107 49 35
1 78 98 73 99
1 19 3 33 16
1 90 82 9 55
1 69 71 59 49
1 90 30 33 0
1 62 84 47 17
1 26 108 33 121
1 65 54 55 83
1 53 7 91 85
1 60 72 99 71
1 35 12 12 74
1 31 96 37 75
1 69 63 43 95
1 1 102 57 86
1 88 96 42 73
1 99 17 52 96
1 95 21 3 126
1 75 50 100 111
1 96 27 1 102
1 42 82 57 5
1 41 30 13 115
1 24 91 90 21
1 82 26 38 38
1 80 21 59 105
1 17 64 41 7
1 98 32 65 100
1 57 35 92 113
1 37 88 32 10
1 49 110 81 57
1 10 10 37 44
1 62 123 58 48
1 67 15 42 22
1 30 72 56 48
1 30 13 96 25
1 62 120 23 43
1 25 7 11 44
1 100 94 24 53
1 100 83 53 129
1 99 59 88 119
1 49 25 28 97
1 59 86 92 123
1 47 109 42 34
1 14 42 66 101
1 27 48 39 62
1 37 52 28 0
1 69 51 49 101
1 11 11 22 37
1 18 125 9 49
1 91 126 64 79
1 80 8 44 49
1 96 28 5 112
1 69 29 34 94
1 99 76 6 45
1 81 11 76 122
1 14 67 89 59
1 58 36 23 92
1 35 36 96 15
1 86 89 44 46
1 1 98 64 102
1 33 58 23 32
1 83 126 60 44
1 66 14 21 62
1 68 105 28 118
1 83 17 50 116
1 4 33 52 38
1 89 5 90 123
1 7 54 41 103
1 75 93 39 105
1 11 107 5 108
1 51 55 84 13
1 52 82 49 18
1 70 16 9 96
1 34 41 12 112
1 85 25 91 78
1 15 81 86 114
1 8 115 77 51
1 44 36 18 16
1 75 99 96 79
1 24 56 28 107
1 63 7 58 4
1 50 117 11 8
1 48 123 41 83
1 65 25 86 112
1 78 36 92 113
1 29 98 87 17
1 56 34 2 9
1 55 108 19 106
1 25 129 34 27
1 27 95 59 62
1 81 99 96 80
1 52 25 30 124
1 61 28 2 22
1 2 57 50 38
1 3 86 20 115
1 84 48 17 83
1 53 3 87 118
1 97 118 32 67
1 20 108 56 110
1 16 85 49 120
1 12 8 32 24
1 34 106 62 37
1 23 85 74 65
1 100 35 35 98
1 35 101 40 2
1 4 7 8 39
1 20 50 4 10
1 46 126 70 58
1 92 42 99 115
1 96 107 12 77
1 49 8 56 45
1 78 112 20 109
1 34 45 77 4
1 29 36 15 100
1 52 29 64 28
1 3 54 86 18
1 78 16 82 117
1 21 83 78 37
1 63 81 42 0
1 72 53 98 129
1 25 103 15 7
1 43 119 17 106
1 7 44 10 35
1 2 68 82 117
1 5 41 82 128
1 10 123 28 65
1 64 127 7 81
1 26 119 82 23
1 48 96 63 12
1 20 59 71 35
1 86 24 58 36
1 65 78 10 50
1 22 127 79 27
1 67 120 68 79
1 69 81 14 13
1 94 125 80 86
1 54 43 69 99
1 93 83 2 89
1 43 7 64 37
1 24 18 46 112
1 95 115 47 86
1 83 48 95 62
1 16 75 98 84
1 89 69 99 105
1 11 105 9 123
1 91 6 5 84
1 42 47 83 109
1 53 51 33 10
1 67 71 99 36
1 71 129 22 17
1 60 4 7 31
1 23 4 29 55
1 7 85 59 4
1 84 94 48 13
1 97 123 81 2
1 33 102 5 109
1 30 70 47 78
1 66 33 48 31
1 47 36 40 28
1 43 13 49 94
1 83 15 26 99
1 94 75 75 107
1 23 128 76 112
1 88 19 38 82
1 90 83 81 60
1 65 21 1 26
1 53 62 86 112
1 22 56 45 98
1 47 56 33 55
1 19 11 38 61
1 31 56 5 8
1 28 5 43 29
1 50 18 25 72
1 55 36 35 123
1 11 21 86 17
1 89 67 19 31
1 13 50 87 24
1 86 73 73 34
1 61 82 86 13
1 18 3 59 40
1 34 88 89 112
1 14 88 98 38
1 37 102 30 119
1 2 0 93 97
1 47 107 31 53
1 25 22 34 124
1 38 107 25 26
1 52 49 24 110
1 9 125 76 79
1 1 11 8 53
1 87 47 41 64
1 34 130 19 74
1 57 96 92 125
1 66 74 58 122
1 99 72 57 57
1 10 86 47 96
1 81 124 53 116
1 58 94 97 89
1 33 77 42 27
1 89 19 57 53
1 7 32 82 105
1 31 126 51 96
1 36 46 42 36
1 75 29 29 10
1 90 9 44 37
1 94 126 90 92
1 14 71 20 66
1 2 17 4 46
1 11 87 93 6
1 54 28 22 0
1 59 71 8 3
1 92 123 77 29
1 42 9 14 104
1 99 110 46 79
1 56 68 59 121
1 21 51 85 108
1 1 18 74 84
1 68 114 98 54
1 67 102 95 46
1 73 76 18 24
1 25 41 47 104
1 73 72 57 0
1 92 33 43 34
1 17 39 10 42
1 94 121 19 4
1 14 100 98 38
1 70 38 42 21
1 28 130 51 9
1 43 99 48 129
1 98 76 81 78
1 4 23 87 2
1 20 36 94 58
1 41 61 98 84
1 33 21 45 121
1 5 106 13 36
1 19 71 79 70
1 34 37 39 82
1 38 99 92 95
1 98 110 11 76
1 66 82 98 49
1 6 28 37 102
1 98 95 8 48
1 31 89 6 67
1 54 90 89 50
1 81 60 1 61
1 78 87 69 83
1 100 44 3 71
1 17 58 75 59
1 90 73 79 99
1 36 48 79 34
1 24 92 93 113
1 31 82 77 52
1 34 49 17 46
1 92 58 1 92
1 97 8 94 59
1 22 8 45 38
1 59 123 49 29
1 62 118 70 104
1 91 67 59 15
1 79 76 2 14
1 42 102 7 77
1 68 114 33 62
1 53 18 77 71
1 5 130 75 44
1 61 75 81 39
1 78 128 40 125
1 41 70 76 86
1 37 100 22 65
1 9 65 56 56
1 41 100 39 95
1 39 54 32 74
1 60 51 77 26
1 79 130 8 4
1 41 75 61 82
1 34 15 22 115
1 7 107 69 43
1 20 95 27 102
1 17 44 99 92
1 19 74 49 94
1 8 39 66 51
1 74 71 28 29
1 47 65 77 83
1 27 46 7 15
1 16 79 15 48
1 89 54 64 70
1 49 65 8 67
1 88 110 71 66
1 93 107 64 10
1 77 98 73 78
1 30 119 26 24
1 23 84 71 13
1 46 80 76 33
1 24 15 74 0
1 16 23 92 127
1 49 38 83 76
1 59 98 7 51
1 60 15 46 109
1 66 60 81 73
1 4 34 26 16
1 71 90 8 112
1 93 6 54 52
1 73 126 38 44
1 24 48 9 55
1 39 57 7 29
1 21 111 58 40
1 12 25 29 51
1 11 78 54 25
1 77 103 34 61
1 70 67 87 46
1 74 2 13 5
1 90 95 26 19
1 79 22 80 17
1 87 84 64 52
1 96 42 31 10
1 39 35 15 51
1 70 17 87 44
1 96 67 40 30
1 58 7 39 99
1 18 9 26 106
1 51 84 70 126
1 37 80 66 115
1 77 30 43 17
1 81 123 58 2
1 30 103 35 57
1 52 35 90 43
1 74 103 15 122
1 6 81 68 79
1 25 87 59 76
1 12 41 25 40
1 99 11 44 105
1 56 47 63 108
1 61 102 75 34
1 50 1 53 55
1 6 4 8 122
1 90 1 88 113
1 8 23 61 4
1 28 33 88 15
1 44 37 60 56
1 42 107 76 31
1 9 111 8 56
1 26 67 2 28
1 98 34 32 76
1 94 7 81 99
1 76 98 99 114
1 45 20 65 60
1 12 60 98 104
1 88 19 50 67
1 62 29 51 79
1 13 0 47 120
1 3 30 99 118
1 52 130 27 22
1 69 38 70 22
1 49 82 98 92
1 76 5 45 36
1 28 67 62 64
1 55 62 74 52
1 69 91 40 39
1 69 8 18 118
1 76 61 59 111
1 97 10 35 86
1 32 85 99 22
1 62 98 60 95
1 13 44 49 42
1 78 22 99 99
1 17 79 94 86
1 47 98 15 27
1 12 127 37 20
1 2 25 40 29
1 94 63 74 103
1 44 52 58 124
1 41 6 95 35
1 8 117 62 102
1 21 8 39 111
1 2 15 63 19
1 27 67 82 101
1 65 5 13 41
1 43 100 9 42
1 18 119 50 63
1 33 121 41 16
1 59 27 95 3
1 36 11 66 94
1 24 19 67 125
1 48 54 85 74
1 79 69 12 1
1 39 127 92 102
1 18 77 47 11
1 69 59 83 60
1 13 122 72 6
1 21 44 11 4
1 71 2 47 65
1 8 32 23 90
1 19 46 43 7
1 31 95 35 26
1 51 25 39 8
1 95 108 50 38
1 42 103 95 65
1 14 110 93 41
1 26 61 62 71
1 13 28 4 37
1 57 40 51 42
1 71 21 73 16
1 70 43 25 46
1 26 118 6 58
1 4 28 10 73
1 84 14 49 8
1 73 17 17 127
1 43 38 29 86
1 45 26 66 41
1 86 21 36 93
1 79 79 62 128
1 56 52 36 2
1 72 87 65 99
1 96 34 55 27
1 49 101 72 80
1 97 103 18 37
1 53 15 6 65
1 24 41 43 51
1 1 41 90 61
1 24 70 38 39
1 58 123 72 119
1 26 43 93 114
1 30 123 90 20
1 40 123 1 73
1 28 101 16 16
1 26 52 6 32
1 11 7 51 25
1 12 86 34 3
1 66 91 19 110
1 13 58 49 21
1 76 32 68 113
1 34 0 10 115
1 5 64 60 33
1 97 87 38 120
1 11 36 32 98
1 91 51 39 49
1 86 114 46 58
1 55 83 72 39
1 37 39 83 130
1 77 69 22 124
1 14 45 54 125
1 13 120 68 77
1 57 40 25 124
1 50 111 63 0
1 99 30 16 111
1 14 122 58 71
1 1 113 9 83
1 100 55 39 13
1 77 52 11 37
1 39 94 94 81
1 13 15 79 19
1 79 30 74 126
1 39 130 52 97
1 32 61 36 37
1 74 91 31 86
1 80 71 5 75
1 100 93 93 82
1 18 107 82 9
1 68 55 3 12
1 4 62 65 71
1 36 89 84 129
1 56 19 61 53
1 88 38 59 28
1 65 96 14 53
1 68 125 91 62
1 22 95 7 32
1 70 34 80 117
1 48 11 31 44
1 66 53 42 48
1 90 46 8 94
1 60 116 29 92
1 93 35 25 123
1 5 39 72 20
1 55 123 31 74
1 87 90 88 85
1 99 106 23 23
1 92 122 98 104
1 70 73 7 41
1 99 59 41 75
1 46 125 39 87
1 61 15 77 87
1 33 68 14 61
1 82 98 86 28
1 54 84 95 69
1 32 77 62 75
1 32 90 13 78
1 39 76 8 48
1 28 112 6 32
1 15 61 3 25
1 17 40 76 16
1 84 15 57 108
1 2 74 52 76
1 32 60 18 94
1 66 114 68 67
1 13 75 52 30
1 75 63 50 43
1 13 57 45 20
1 4 130 61 33
1 49 71 53 9
