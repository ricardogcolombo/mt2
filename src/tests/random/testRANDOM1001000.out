0.592593
0.272727
0.521739
0.434783
0.615385
0.375000
0.368421
0.631579
0.636364
0.470588
0.363636
0.437500
0.529412
0.250000
0.722222
0.363636
0.478261
0.541667
0.500000
0.631579
0.481481
0.333333
0.529412
0.631579
0.583333
0.608696
0.363636
0.785714
0.545455
0.454545
0.666667
0.500000
0.466667
0.458333
0.352941
0.555556
0.400000
0.631579
0.476190
0.500000
0.421053
0.434783
0.454545
0.562500
0.466667
0.416667
0.583333
0.500000
0.444444
0.565217
0.454545
0.777778
0.578947
0.640000
0.571429
0.437500
0.631579
0.653846
0.611111
0.291667
0.464286
0.400000
0.478261
0.440000
0.687500
0.470588
0.625000
0.466667
0.538462
0.357143
0.434783
0.500000
0.296296
0.478261
0.500000
0.480000
0.450000
0.368421
0.500000
0.562500
0.666667
0.461538
0.384615
0.500000
0.500000
0.500000
0.384615
0.500000
0.565217
0.450000
0.619048
0.538462
0.263158
0.500000
0.444444
0.450000
0.533333
0.285714
0.647059
0.653846
